src/components/Movies.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
 import Poster from './Poster';
+import scrollLeftButton from './scroll-left.png';
+import scrollRightButton from './scroll-right.png';
 import './Movies.css';
 
 const Movies = ({movies}) => {
 
+  const scroller = React.useRef();
+
+  const scrollLeft = () => {
+    console.log(scroller.current);
+    scroller.current.scrollLeft -= 600;
+  }
+
+  const scrollRight = () => {
+    scroller.current.scrollLeft += 600;
```

Comment:
Do these number still do the same amount even if the window is bigger? I think these are a measurement of pixels, so if the window was say 300px wide, would it skip basically a whole section?